Разметка и стилизация

kseli opened the pull request.

kseli committed:
Добавляет наставника

kseli committed:
разметка

kseli committed:
разметка и стилизация

kseli committed:
Стилизация главной страницы

AlSudar commented on catalog.html:
```diff
@@ -0,0 +1,465 @@
+              <a class="navigation-link catalog-link" href="#">
+                Каталог
+              </a>
+              <ul class="submenu">
```
Наверное пока что стоит скрыть это меню

AlSudar commented on catalog.html:
```diff
@@ -0,0 +1,465 @@
+            <li class="product-list-item">
+              <a class="product-list-link" href="#">
+                <img class="product-img" src="images/raspberry.png" alt="" width="168" height="168">
+                <h2 class="product-title">Малинка</h2>
```
Мне кажется тут логичнее использовать заголовок третьего уровня, т.к. заголовок второго уровня - это заголовок секции

AlSudar commented on catalog.html:
```diff
@@ -0,0 +1,465 @@
+
+          <div class="pagination-wrapper">
+            <button class="button button-light button-show-more" type="button">Показать еще</button>
+            <ul class="pagination">
```
Тут лучше использовать ol, т.к. порядок элементов списка важен

AlSudar commented on index.html:
```diff
@@ -1,12 +1,406 @@
+        <div class="about-us-wrapper">
+          <section class="about-us">
+            <h2 class="visually-hidden">О нас.</h2>
+            <p class="section-title about-us-title">Магазин Глейси – это онлайн и офлайн-магазин по продаже мороженого
```
Мне кажется это должен быть заголовок вместо visually-hidden

AlSudar commented on index.html:
```diff
@@ -1,12 +1,406 @@
+              производства на развес</p>
+            <ul class="about-us-list">
+              <li class="about-us-item ice-cream-icon">
+                <p class="about-us-text">Всё наше мороженое изготавливается на собственном производстве с
```
Это лишняя обёртка

kseli replied:
@AlSudar не очень поняла, какая обертка?

kseli replied:
div.about-us-wrapper? Я сделала ее для подложки-вафельки на которой background-image и background-repeat

AlSudar replied:
Нет, .about-us-text - вот эта. Текст можно вставить сразу в li, а иконку добавить как псевдоэлемент

AlSudar commented on styles/styles.css:
```diff
@@ -0,0 +1,920 @@
+  cursor: pointer;
+}
+
+.button:hover {
```
Чтобы добавить плавности и красоты эффектам можешь попробовать добавить свойства transition

kseli committed:
стилизация

kseli committed:
стилизую каталог

AlSudar commented on index.html:
```diff
@@ -272,8 +282,10 @@ <h2 class="visually-hidden">О нас.</h2>
-            <h3 class="news-theme">10 способов сервировки фруктовых щербетов к&nbsp;столу</h3>
+            <a class="news-link" href="#">
+              <h2 class="news-title">Новое в нашем блоге</h2>
+              <h3 class="news-theme">10 способов сервировки фруктовых щербетов к&nbsp;столу</h3>
```
Мне кажется это должен быть параграф, а не заголовок. Даже с учётом того что тут размер шрифта больше чем у заголовка, всё равно это скорее описание, чем заголовок

kseli replied:
Разве это не заголовок статьи?

AlSudar replied:
Даже если это заголовок статьи, это же не заголовок этого блока

AlSudar commented on styles/styles.css:
```diff
@@ -306,17 +319,37 @@ body {
-  padding: 21px 0 0 20px;
+.slider-right-column {
   position: relative;
+  display: flex;
```
Мне кажется что в slider-button свойство position: relative лишнее

kseli committed:
стилизую каталог..

kseli committed:
делаю всплывающие окна на главной

kseli committed:
поповер личный кабинет

kseli committed:
поповеры, состояния и немного правок

kseli committed:
модальное окно готово

kseli committed:
меняю чекбокс

Все почти готово. Остается только подготовить к защите и сделать состояния фокуса для чекбоксов

AlSudar commented:
В общем и целом всё хорошо, проект уже почти готов к защите, осталось поправить несколько замечаний: 
При переполнении ломается блок user-menu-wrapper
При переполнении блока news фоновое изображение слишком растягивается
Для control-mark нужно добавить свойство box-sixing: border-box, чтобы ширина их рамок не учитывалась в конечном размере радиокнопок

kseli committed:
правки внесены

осталась финальная подготовка по критериям

kseli committed:
подготовка к защите

kseli committed:
на получение допуска
